docs/source/tutorials/qmd/03-gev.qmd

```diff
@@ -59,7 +59,7 @@ model <- liesel(
 )
 ```
 
-Now, we can continue in Python and use the `lsl.dist_reg_mcmc()` function to set up a sampling algorithm with IWLS kernels for the regression coefficients ($\boldsymbol{\beta}$) and a Gibbs kernel for the smoothing parameter ($\tau^2$) of the spline. Note that we need to set $\beta_0$ for $\xi$ to 0.1 manually, because $\xi = 0$ breaks the sampler.
+Now, we can continue in Python and use the `lsl.dist_reg_mcmc()` function to set up a sampling algorithm with IWLS kernels for the regression coefficients ($\boldsymbol{\beta}$) and a Gibbs kernel for the smoothing parameter ($\tau^2$) of the spline. Note that we need to set $\beta_0$ for $\xi$ to 0.1 manually, because $\xi = 0$ breaks the sampler. In order to avoid numerical instabilty and consequently invalid supports, we do not appy any jittering to $\boldsymbol{\beta}$ and $\tau^2$.
```

Comment:
The reader who doesn't know about the support of the GEV distribution might be confused. I think the support varying with parameters is not mentioned in the tutorial. I would write something like

> The support of the GEV distribution changes with the parameter values (compare [Wikipedia](https://en.wikipedia.org/wiki/Generalized_extreme_value_distribution)). To ensure that the initial parameters, support the observed data we set xi = 0 and disable jittering of the the variance and regression parameters. For the latter, we supply functions user-defined jitter functions to `lsl.dist_reg_mcmc` that are essentially the identity function wrt the parameter value.

Reply:
Ok. Thanks.